### PR TITLE
Feature/remove debug stmts

### PR DESCRIPTION
**Pull request by jdelamare**
This branch would have been more appropriately named `feature/optimized-build`.  The Dockerfile has the correct command to build more compact code, which is then copied over to the webserver's directory. `Debug` statements needed to be removed so that this could happen. 

If this needed to be tested, I'd say having two versions of the container system, one with 
```
    && npm run build:staging \
``` 
and one with
```
    && npm run build \
```
in `Dockerfile.node` would do the trick. One would then ssh into the server, exec onto the container, and check the file sizes of each build for comparison.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "Frontend for the ereadingtool",
   "scripts": {
-    "start": "npm install && npm run build:dev && npm run dev",
+    "start": "npm install && npm run build && npm run",
```
Comment (bgins) [inline web/package.json:6]: Can we keep these set to `dev`? We use these for local development to keep parcel running.

**Review by bgins — CHANGES_REQUESTED** [review]
Let's keep the config in `pacakge.json` if we can. Don't think it should effect the build on the server.

File: web/package.json (hunk 1 above)
Comment (jdelamare) [reply web/package.json]: Agreed

**Review by bgins — APPROVED** [review]
Looks good!

**Pull request merged**